src/coreclr/jit/emitarm64.h

```diff
@@ -890,6 +886,18 @@ void emitIns_S_R(instruction ins, emitAttr attr, regNumber ireg, int varx, int o
 void emitIns_S_S_R_R(
     instruction ins, emitAttr attr, emitAttr attr2, regNumber ireg, regNumber ireg2, int varx, int offs);
 
+void emitIns_SS_R_R_R_I(instruction ins,
```

Comment:
`emitIns_R_R_S_S()` and `emitIns_R_S()` already exist.

Should `emitIns_SS_R_R_R_I()` be `emitIns_R_R_R_I_S_S()` ?

(I'm a little uncertain on how the naming scheme works here).

Reply:
The naming varies slightly in different architectures, but in this case, it is `emitIns_Dst_Src`, so `emitIns_R_R_S_S()` means "instruction loading from stack to register" which is `ldp` and `emitIns_S_S_R_R()` is an instruction that loads from register to stack i.e. `stp`. 

Glad that you pointed out, but in this case since `emitIns_SS_R_R_R_I()` handles both `ldp` and `stp`, I should just name it as `emitIns_R_R_R_I_LdStPair()`.